ICD10/manual_work/crm_changes.sql

```diff
@@ -13,10 +13,78 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 * 
-* Authors: Darina Ivakhnenko, Dmitry Dymshyts
+* Authors: Irina Zherko, Darina Ivakhnenko, Dmitry Dymshyts
 * Date: 2021
 **************************************************************************/
+-- create current date backup of concept_relationship_manual table
+DO
+$body$
+    DECLARE
+        update text;
+    BEGIN
+        SELECT TO_CHAR(CURRENT_DATE, 'YYYY_MM_DD')
+        INTO update;
+        EXECUTE format('create table %I as select * from concept_relationship_manual',
+                       'concept_relationship_manual_backup_' || update);
+
+        END
+$body$;
+
 -- deprecate previous inaccurate mapping
-UPDATE concept_relationship_manual SET valid_end_date = CURRENT_DATE, invalid_reason = 'D' WHERE concept_code_1 IN (SELECT icd_code FROM refresh_lookup_done) AND concept_code_2 NOT IN (SELECT repl_by_code FROM refresh_lookup_done);
+UPDATE concept_relationship_manual crm
+SET invalid_reason = 'D',
+    valid_end_date = current_date
+
+--SELECT * FROM concept_relationship_manual crm --use this SELECT for QA
+WHERE invalid_reason IS NULL --deprecate only what's not yet deprecated in order to preserve the original deprecation date
+
+    AND concept_code_1 IN (SELECT icd_code FROM refresh_lookup_done) --work only with the codes presented in the manual file of the current vocabulary refresh
+
+    AND NOT EXISTS (SELECT 1 --don't deprecate mapping if the same exists in the current manual file
+                    FROM refresh_lookup_done rl
```

Comment:
where is this table stored?

Reply:
refresh_lookup_done table should be created at every refresh